RecoTracker/LST/src/alpaka/ES_ModulesDev.cc

```diff
@@ -1,7 +1,10 @@
-#include <SDL/Module.h>
+#include <SDL/LST.h>
 #include "HeterogeneousCore/AlpakaCore/interface/alpaka/typelookup.h"
 
 // Temporary hack: The DevHost instantiation is needed in both CPU and GPU plugins,
 // whereas the (non-host-)Device instantiation only in the GPU plugin
-TYPELOOKUP_DATA_REG(SDL::modulesBuffer<alpaka_common::DevHost>);
-TYPELOOKUP_ALPAKA_TEMPLATED_DATA_REG(SDL::modulesBuffer);
+TYPELOOKUP_DATA_REG(SDL::LSTESHostData<SDL::Dev>);
+TYPELOOKUP_DATA_REG(SDL::LSTESDeviceData<SDL::DevHost>);
+TYPELOOKUP_DATA_REG(ALPAKA_ACCELERATOR_NAMESPACE::ESDeviceProduct<std::unique_ptr<SDL::LSTESHostData<SDL::Dev>>>);
```

Comment:
- is this needed?
    - if yes, is this really supposed to be `std::unique_ptr` instead of the underlying type?  (looking at `ESDeviceProduct` use cases, it's a somewhat internally used type by the template argument seems to be implying the product data, not a `unique_ptr` of it

Reply:
Yeah, this was the only way I was able to get the linker to find all the symbols. There's probably a better way to do it.